ConsoleApp1/ResultsData.cs

```diff
@@ -16,7 +16,7 @@ class JsonResultsData
         public static JsonResultsData? Load(string path)
         {
             var options = new JsonSerializerOptions { IncludeFields = true };
-            return JsonSerializer.Deserialize<JsonResultsData>(File.ReadAllText(path), options);
+            return JsonSerializer.Deserialize<JsonResultsData>(path, options);
```

Comment:
I think we should keep the Load method as it was before and introduce new method, which might looks like
`public static JsonResultsData? Load(string url, string json) { ... }`, which would deserialize the json string into `JsonResultsData`.

For the `FlavorData` we would need new constructor, which can look like:
`public FlavorData(string baseUrl, string flavor, string resultsJson, string gitlog) { ... }`. It would also need another `LoadGitLog` method, modified in similar way.